# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1208397059 14400
#      Wed Apr 16 21:50:59 2008 -0400
# Node ID 62279ce5654c95c7ce32dcea8b45b444fbd877ef
# Parent  ff918ee1a9830e3f23c4f281bf02f6945faa4bce
save: fix continuation character in doc string

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,10 +1,12 @@
 2008-04-16  John W. Eaton  <jwe@octave.org>
 
+	* load-save.cc (Fsave): Fix continuation character in doc string.
+
 	* pt-walk.h (tree_walker::visit_function_def): New function.
 	* pt-pr-code.cc (tree_print_code::visit_function_def): New function.
 	* pt-pr-code.h: Provide decl.
 	* pt-bp.cc (tree_breakpoint::visit_function_def): New function.
 	* pt-bp.h: Provide decl.
 	* pt-check.cc (tree_checker::visit_function_def): New function.
 	* pt-check.h: Provide decl.
 
diff --git a/src/load-save.cc b/src/load-save.cc
--- a/src/load-save.cc
+++ b/src/load-save.cc
@@ -1495,17 +1495,17 @@ Match any single character.\n\
 @item *\n\
 Match zero or more characters.\n\
 \n\
 @item [ @var{list} ]\n\
 Match the list of characters specified by @var{list}.  If the first\n\
 character is @code{!} or @code{^}, match all characters except those\n\
 specified by @var{list}.  For example, the pattern @samp{[a-zA-Z]} will\n\
 match all lower and upper case alphabetic characters. \n\
-\n\      
+\n\
 Wildcards may also be used in the field names specifications when using\n\
 the @code{-struct} modifier (but not in the struct name itself).\n\
 \n\
 @item -text\n\
 Save the data in Octave's text data format.\n\
 @end table\n\
 \n\
 Except when using the @sc{Matlab} binary data file format, saving global\n\
